deploy/examples/import-external-cluster.sh

```diff
@@ -96,11 +96,11 @@ function checkEnvVars() {
 }
 
 function importClusterID() {
-  if [ -n "$RADOS_NAMESPACE_CLUSTER_ID" ]; then
```

Comment:
I thought the cluster ID variable was different from an actual namespace. Are you sure this is the right variable to use? Did you find this issue, or where is the source of the bug?

Reply:
> I thought the cluster-ID variable was different from an actual namespace

Yes, it is different if the user wanted to create `To create CephFS PVC in a subvolumeGroup`  or `To create RBD PVC in a RadosNamespaces` The cluster-Id should be changed to respective id's,
Otherwise, it would be set to `CLUSTER_NAMESPACE`


> Are you sure this is the right variable to use

Yes because you can see what we are exporting here `SUBVOLUME_GROUP` https://github.com/rook/rook/blob/37758836cac0d32c7bfeb9106813d143933ad5f8/deploy/examples/create-external-cluster-resources.py#L1330

>Did you find this issue, or where is the source of the bug

I was updating the doc downstream and checking if these flags need to be updated there, but the use is purely upstream for now till now no such requirement is there downstream, and realized, oh the variable I am using is not right.

Reply:
Ok thanks for the explanation. I see we're not using these vars anywhere else, so it makes sense to use the same ones exported from the other script.